scripts/sanity/guard-controller/base-test.cjs

```
@@ -595,18 +917,45 @@ class BaseGuardControllerTest {
         
         switch (actionType) {
+            case this.RoleConfigActionType.CREATE_ROLE:
+                // (string roleName, uint256 maxWallets, FunctionPermission[] functionPermissions)
+                // FunctionPermission is tuple(bytes4,uint16,bytes4[])
+                const permTuples = (data.functionPermissions || []).map(p => [
+                    p.functionSelector,
+                    typeof p.grantedActionsBitmap === 'number' ? p.grantedActionsBitmap : this.createBitmapFromActions(p.actions || []),
+                    p.handlerForSelectors || [p.functionSelector]
+                ]);
+                encodedData = this.web3.eth.abi.encodeParameters(
+                    ['string', 'uint256', 'tuple(bytes4,uint16,bytes4[])[]'],
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)